tools/make/helm.mk

```diff
@@ -27,3 +27,7 @@ helm-install: helm-generate ## Install envoy gateway helm chart from OCI registr
 
 helm-generate:
 	ImageRepository=${IMAGE} ImageTag=${TAG} envsubst < charts/gateway-helm/values.tmpl.yaml > ./charts/gateway-helm/values.yaml
+
+lint: helm-lint
+helm-lint: helm-generate
```

Comment:
looking at our CI
https://github.com/envoyproxy/gateway/blob/main/.github/workflows/build_and_test.yaml
it looks like `helm-generate` is directly called
should we just move the `helm lint` command as a step inside `helm-generate` ?

Reply:
According to your recommendation, it is completely okay in ci. but I think `helm lint` should be a part of `make lint` Otherwise, if the user executes `make lint` separately, there will be no`helm lint` step.

Reply:
yeah instead it will get triggered in `helm generate` step and fail that step which imo is fine.
If we include `helm lint` in `lint` we will end up calling `helm generate` multiple times

Reply:
make sense. Already updated.